main/lib/session.js

```diff
@@ -604,8 +604,20 @@ export class HttpWTSession {
         args.error
     )
 
-    this.sendStreamsController.forEach((ele) => ele.error(wtError))
-    this.receiveStreamsController.forEach((ele) => ele.error(wtError))
+    this.sendStreamsController.forEach((ele) => {
+      try {
+        ele.error(wtError)
+      } catch {
+        // Controller already errored/closed - expected during cleanup
+      }
+    })
+    this.receiveStreamsController.forEach((ele) => {
+      try {
+        ele.error(wtError)
+      } catch {
+        // Controller already errored/closed - expected during cleanup
+      }
+    })
```

Comment:
Same as above. This just hide problems. I am happy to fix problems, but not by just hiding the problems.